### ISSUE TITLE
Some tiles on conveyor belts are blocked

### ISSUE THREAD

**Issue by dethstryke01** [OpenApoc/OpenApoc#1326]
Some tiles on conveyor belts are "blocked," and pathfinding also seems to become extremely confused around them. 

[https://youtu.be/BRogQm5x4Iw?t=4150](url)

Window 10 x64 22H2
OpenApoc version 0.0.930
no mods in use

[settings.zip](https://github.com/OpenApoc/OpenApoc/files/14152413/settings.zip)

[save_test 3.zip](https://github.com/OpenApoc/OpenApoc/files/14152414/save_test.3.zip)

**Comment by ayrtondenner** [comment]
Regarding the second video, this seems to be correct, since these blocked tiles have entities on them: its just these entities are rendered in level 3, but your camera was in level 2.

![image](https://github.com/OpenApoc/OpenApoc/assets/13112588/9f8745dc-cc90-4f37-b7da-a3a2c1bc085c)
![image](https://github.com/OpenApoc/OpenApoc/assets/13112588/205e06d6-ed48-4168-8877-401f175557d3)

What I found weird is this path order, where mouse hover shows the shortest path, but actually clicking sent agent trough a longer route:

![image](https://github.com/OpenApoc/OpenApoc/assets/13112588/a92a5585-c471-4d86-861d-feddb603c153)
![image](https://github.com/OpenApoc/OpenApoc/assets/13112588/a45658a6-1075-41de-abe8-fb28093b1605)

**Comment by dethstryke01** [comment]
You're right, I didn't notice those plastic sculptures or whatever they are on the next level. Pathfinding becomes extremely wonky around the conveyor belts, and it seems to prefer to take any possible path to avoid them. Not sure what's going on with that.
